app/printPairs.js

```diff
@@ -0,0 +1,27 @@
+const MajorColorNames = ["WHITE", "RED", "BLACK", "YELLOW", "VIOLET"];
+const MinorColorNames = ["BLUE", "ORANGE", "GREEN", "BROWN", "SLATE"];
+
+function ColorPair() {
+  this.majorColor;
+  this.minorColor;
+}
+ColorPair.prototype.toString = function () {
+  return `MajorColor:${this.majorColor},MinorColor:${this.minorColor}`;
+};
+function printColorPairs() {
+  console.log("Color Code and Pairs Manual");
+  let colorCoding = "";
+  for (let i = 0; i < MajorColorNames.length; i++) {
+    for (let j = 0; j < MinorColorNames.length; j++) {
+      let pairNumber = i * MinorColorNames.length + (j + 1);
+      colorCoding += `Pair Number: ${pairNumber}, MajorColor: ${MajorColorNames[i]}, MinorColor: ${MinorColorNames[j]}\n`;
+    }
+  }
+  return colorCoding;
```

Comment:
Good approach to separate the formatting of the manual from its console log :+1: 
How would you prove its functionality? Think of a customer who would use this manual. Write the test in such a way - if it passes, we should be confident that the customer would be satisfied